apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/OperationTasksBuilder.java

```diff
@@ -130,26 +134,29 @@ private List<TableDataLayoutMetadata> rankAndSelectFromTableDataLayoutMetadataLi
         "Total estimated compute cost: {}, total estimated reduced file count: {}",
         totalComputeCost,
         totalReducedFileCount);
-    meter
-        .counterBuilder("data_layout_optimization_estimated_compute_cost")
-        .build()
-        .add((long) totalComputeCost);
-    meter
-        .counterBuilder("data_layout_optimization_estimated_reduced_file_count")
-        .build()
-        .add((long) totalReducedFileCount);
+    otelEmitter.count(
+        METRICS_SCOPE,
+        "data_layout_optimization_estimated_compute_cost",
+        (long) totalComputeCost,
+        null);
```

Comment:
Good to add attribute here as well for consistency. This can be added later as the name itself tells dlo metrics.